edx_lint/files/pylintrc

```diff
@@ -36,28 +249,88 @@ load-plugins=edx_lint.pylint,pylint_django,pylint_celery
 # no Warning level messages displayed, use"--disable=all --enable=classes
 # --disable=W"
 disable=
+    # These should be left to the discretion of the reviewer
+    bad-continuation,
+    invalid-name,
+    misplaced-comparison-constant,
+    file-ignored,
+    too-many-nested-blocks,
+    too-many-statements,
```

Comment:
`too-many-statements` seems like a good one to keep.  It violates our Clean Code initiative.  And if we ever run into it, it's a good time to refactor the function.  (If anything, the number of acceptable lines in a method could be reduced.)